chainercv/links/detection_link.py

```diff
@@ -0,0 +1,39 @@
+import chainer
+
+
+class DetectionLink(chainer.Link):
+    """A chainer.Link for object detection.
+
+    This is an abstract class for object detection links.
+    All object detectors should inherit this class.
+    """
+
+    def predict(self, img):
+        """Detect objects in an image.
+
+        This method detects objects in an image.
+        This method returns a tuple of three arrays,
+        :obj:`(bbox, label, score)`.
+
+        Args:
+            img (~numpy.ndarray): An array of shape :math:`(3, height, width)`.
+                This is in BGR format and the range of its value is
+                :math:`[0, 255]`.
```

Comment:
I know we sometimes want to pass a batch to a model for speed.
How about `[(bbox, label, score), (bbox, label, score)...]`?